--- /home/dzliu/Software/CONDA/miniconda3/envs/jwstpmap1075/lib/python3.9/site-packages/jwst/tweakreg/tweakreg_step.py.ori  2023-05-03 02:48:44.337989456 +0200
+++ /home/dzliu/Software/CONDA/miniconda3/envs/jwstpmap1075/lib/python3.9/site-packages/jwst/tweakreg/tweakreg_step.py  2023-05-03 02:50:25.349357917 +0200
@@ -124,7 +124,11 @@
                         elif 'tweakreg_catalog' in member:
                             del member['tweakreg_catalog']

-                    images.from_asn(input)
+                    #<DZLIU># 2023-01-09 BUG???
+                    #<DZLIU># images.from_asn(input)
+                    self.log.info('dzliu fixing a potential tweak_step bug "images.from_asn(input)" --> "images.from_asn(asn_data)"')
+                    images.from_asn(asn_data)
+                    #<DZLIU>#

                 elif is_association(input):
                     images.from_asn(input)
